<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="nada-Vostro-15-3510" ignored="0" name="Surefire test" tests="18" failures="3" timestamp="2022-07-23T01:31:30 EET" time="351.717" errors="0">
  <testcase name="runScenario" time="26.948" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="17.644" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="18.12" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="15.398" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="15.059" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="15.193" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="15.041" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="23.87" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="22.995" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="23.604" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="12.725" classname="io.cucumber.testng.AbstractTestNGCucumberTests">
    <failure type="java.lang.AssertionError" message="expected [https://demo.nopcommerce.com/nokia-lumia-1020] but found [https://demo.nopcommerce.com/]">
      <![CDATA[java.lang.AssertionError: expected [https://demo.nopcommerce.com/nokia-lumia-1020] but found [https://demo.nopcommerce.com/]
at org.example.stepDefs.D06_homeSlidersStepDef.verify1stSlider(D06_homeSlidersStepDef.java:27)
at ✽.user redirect to url1 Nokia(file:///home/nada/IdeaProjects/DemoProjectAutomation2/src/main/resources/features/F06_homeSliders.feature:6)
... Removed 6 stack frames]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase name="runScenario" time="12.8" classname="io.cucumber.testng.AbstractTestNGCucumberTests">
    <failure type="java.lang.AssertionError" message="expected [https://demo.nopcommerce.com/iphone-6] but found [https://demo.nopcommerce.com/]">
      <![CDATA[java.lang.AssertionError: expected [https://demo.nopcommerce.com/iphone-6] but found [https://demo.nopcommerce.com/]
at org.example.stepDefs.D06_homeSlidersStepDef.verify2ndSlider(D06_homeSlidersStepDef.java:34)
at ✽.user redirect to url2 iphone(file:///home/nada/IdeaProjects/DemoProjectAutomation2/src/main/resources/features/F06_homeSliders.feature:10)
... Removed 6 stack frames]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase name="runScenario" time="17.269" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="27.239" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="17.681" classname="io.cucumber.testng.AbstractTestNGCucumberTests">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index: 1, Size: 1">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index: 1, Size: 1
at java.util.ArrayList.rangeCheck(ArrayList.java:659)
at java.util.ArrayList.get(ArrayList.java:435)
at org.example.stepDefs.D07_followUsStepDef.isOpenedInNewTab(D07_followUsStepDef.java:78)
at ✽."https://demo.nopcommerce.com/new-online-store-is-open" is opened in new tab(file:///home/nada/IdeaProjects/DemoProjectAutomation2/src/main/resources/features/F07_followUs.feature:22)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase name="runScenario" time="17.239" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="14.389" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="37.89" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
</testsuite> <!-- Surefire test -->
